README.md

```diff
@@ -773,8 +773,8 @@ A list of **Free** and **Open Source Software** ***(FOSS)*** for **Android** –
 ## – Tools –
 
 * [**Universal Android Debloater GUI**](https://github.com/0x192/universal-android-debloater)
```

Comment:
This line should be moved down I guess.

Reply:
Whoops, yeah, I only ran this script on the - Apps - and - Games - category and I forgot to move that one 😅